style.css

```diff
@@ -512,7 +512,7 @@ body {
     position: absolute;
     height: 61px;
     top: -15px;
-    left: 30px;
+    left: 65px;
     background-color: #d3d3d3;
```

Comment:
- [ ] Dear @SaveryIV, good job trying to address the previous suggestion, but the issue still exists, as you can see from my screenshot, on the `850px` screen size the squares are at the top of your speakers, and on a screen greater than `1600px` your squares are no more link to the image, Please make sure the squares are always fixed with the image and move to where the image is. 

> Please remember, as front-end engineers, we are always expected to clone the given design.

### on-screen >1600px
![image](https://github.com/SaveryIV/First-Capstone-Project/assets/69077061/4deb05c9-ae49-4d01-811a-054edb93e426)
### on-screen >850px
![image](https://github.com/SaveryIV/First-Capstone-Project/assets/69077061/2b56abff-0c7c-447a-9d94-6d936724342a)

Reply:
@TadesseDev and how i can solve the problem? because I tried but I can have the correct result

Reply:
Dear @SaveryIV, This could be achieved in a lot of different ways, but the most easier way I would suggest be to rap the image and the square in one container (`div`) and then add a `position: relative` to the container (`div`) and `position: absolute` to the square, then add the left and top positioning for the element/square. But as I say there are a lot of ways to do this and I would suggest looking for lessons online.